### ISSUE TITLE
Migrate website branch to organization 

### ISSUE THREAD

**Issue by kskarthik** [AppImage/AppImageKit#103]
I'm proposing to move the website branch to appimage organization. I feel that its not a right place for website in AppImageKit repo.

**Comment by probonopd** [comment]
I share this. Will do as soon as I find the time, and when we are doing #78

**Comment by kskarthik** [comment]
If you don't have time. Just create a repo in that org. I will submit a pull request of the site :)

**Comment by probonopd** [comment]
The domain then also needs to be moved over, I need to find out how to do that.

**Comment by chrisnharvey** [comment]
@probonopd Could we not move the entire AppImageKit project to an organisation?

**Comment by probonopd** [comment]
We could and sooner or later we should, but I am a bit hesitant because I don't want break things.

**Comment by kskarthik** [comment]
Better stay as it is now. Github has evolved a lot. You can set admins for use r repos too

**Comment by probonopd** [comment]
I still think sooner or later we should do that...

**Comment by chrisnharvey** [comment]
I'm pretty sure if you move a repository, GitHub sets up a redirect, so everything should still work as it does now, but GitHub will transparently redirect everything to the new location.

I'm not sure how Travis CI will handle the move though, but that should be as simple as enabling the new repository in Travis I think.

Maybe give it a go with a test repo or something?

**Comment by probonopd** [comment]
Also see https://github.com/probonopd/AppImageKit/issues/78

**Comment by azubieta** [comment]
Can we close this ?

**Comment by probonopd** [comment]
No, we have not done this yet since we had no urgent need so far, it is work but the idea is not bad per se. I tend to not do admin work for which there is no urgent need. Please comment or reopen in case urgent need ever arises.

**Comment by TheAssassin** [comment]
I will perform the switch to a separate repository in the near future.

**Comment by probonopd** [comment]
Here is an argument why spreading out stuff is not a good idea: It makes it impossible(?) to send a PR that implements a change and changes the documentation and website along with it.

Code changes, documentation changes, and website changes should be "atomic" in one pull request. Spreading out things into multiple repos makes this much harder if not entirely impossible.

**Comment by TheAssassin** [comment]
That doesn't make any sense at all. The website is maintained in an independent branch. Also, it doesn't contain any data of interest that needed to be updated regularly. So, extracting only removes bloat from the AppImageKit repository.

**Comment by TheAssassin** [comment]
That's not bad. The website doesn't contain _any_ data that is of any interest for developers, that will be needed to updated.

We could also put any software ever made into a single repo...

**Comment by probonopd** [comment]
Right... which is exactly what https://github.com/probonopd/AppImageKit originally was before we started to spread out everything. Sometimes I miss those "easy" days ;-)

**Comment by kskarthik** [comment]
It this still undecided?
